package.json

```diff
@@ -22,7 +22,8 @@
     "lint": "npm run lint:prettier && npm run lint:eslint && npm run lint:ts",
     "prepare": "husky install",
     "release": "npm run build && semantic-release",
-    "test": "jest"
+    "test": "jest",
+    "test:update": "jest --updateSnapshot"
```

Comment:
There is no need for that - `npm run <script> -- <options here>` is the way to pass options - start using it today!
Here it can be `npm test -- -u`.

Reply:
Yep, that does require someone going to check out the jest documentation. First placed I looked was the package.json to see if there was a script to do it, second was the readme. So I figure that adding the script just makes discoverability a bit easier?

Reply:
There's always help for that: `npm test -- --help`.

Working on a project you need to know the tools used. Snapshot testing are Jest's big feature, you learn how to use it once.

For example, it might be better to update snapshots interactively: `npm test -- --watch`, then you get a menu, select <kbd>i</kbd>, and you can inspect and update each snapshot once you check it's correct.